rispy/parser.py

```diff
@@ -180,15 +172,9 @@ def is_counter(self, line):
         return bool(none_or_match)
 
 
-def readris(*args, **kwargs):
-
-    warnings.warn("Please use 'load' instead of 'readris'", FutureWarning)
-
-    load(*args, **kwargs)
-
-
-def load(file, mapping=None, wok=False):
-    """Load a RIS file and return a list of entries.
+def load(file: TextIO, mapping: Optional[Dict] = None, is_wok: bool = False) -> List[Dict]:
+    """
+    Load a RIS file and return a list of entries.
```

Comment:
Docstrings have to start directly after the `"""`. See PEP 257 and Google style guide.